modules/subscriptions.php

```diff
@@ -592,8 +612,11 @@ function widget( $args, $instance ) {
 		$subscribers_total      = $this->fetch_subscriber_count();
 		$widget_id              = esc_attr( !empty( $args['widget_id'] ) ? esc_attr( $args['widget_id'] ) : mt_rand( 450, 550 ) );
 
-		if ( ! is_array( $subscribers_total ) )
-			$show_subscribers_total = FALSE;
+		if ( is_array( $subscribers_total ) && $this->has_show_subscriber_count_option() ) {
```

Comment:
this could be written `$show_subscribers_total = is_array( $subscribers_total ) && $this->has_show_subscriber_count_option();` :)